bundle/edu.gemini.ags.client.api/src/main/scala/edu/gemini/ags/client/api/AgsClient.scala

```diff
@@ -22,7 +21,7 @@ trait AgsClient {
    * Performs an asynchronous estimation request, providing the result to the
    * given callback function when available.
    */
-  def estimate(obs: Observation, time: Long)(callback: Callback) {
+  def estimate(obs: Observation, time: Long)(callback: Callback)(implicit ec: ExecutionContext) {
```

Comment:
Good question.  I didn't look at it much other than to add the implicit parameter.  I guess the idea is that it returns right away and then informs the `Callback` when it is done.  Just having it return a `Future` seems like a better idea to me too but I won't go into refactoring that now.